Jellyfin.Server/Extensions/ApiServiceCollectionExtensions.cs

```diff
@@ -61,6 +62,7 @@ public static IServiceCollection AddJellyfinApiAuthorization(this IServiceCollec
             serviceCollection.AddSingleton<IAuthorizationHandler, IgnoreParentalControlHandler>();
             serviceCollection.AddSingleton<IAuthorizationHandler, FirstTimeOrIgnoreParentalControlSetupHandler>();
             serviceCollection.AddSingleton<IAuthorizationHandler, LocalAccessHandler>();
+            serviceCollection.AddSingleton<IAuthorizationHandler, NetworkAccessHandler>();
```

Comment:
```suggestion
            serviceCollection.AddSingleton<IAuthorizationHandler, LocalNetworkAccessHandler>();
```

Also should update the handler name